src/py/flwr/client/app.py

```diff
@@ -18,9 +18,12 @@
 import argparse
 import sys
 import time
-from logging import INFO
+from logging import INFO, WARN
+from pathlib import Path
 from typing import Callable, ContextManager, Optional, Tuple, Union
 
+import certifi
```

Comment:
@danieljanes @panh99 Maybe I am missing something, but this is not a direct dependency (`requests` for example installs it). We should add it as such if we want to use it (which seems reasonable). It seems like a small utility library so I would be fine with it although

Reply:
@tanertopal @danieljanes I just wanted to know your opinion on using the `certifi` library. Using `certifi` allows us to make minimal modifications to existing code. Alternatively, we could load system certificates bundled with gRPC, but that would require us to add an additional argument, such as enable_https, to specify whether to load system certificates when no certificates are provided. We would need to add enable_https to start_client, connection, and create_channel.

Reply:
Added use_grpc_certificates arg to start_client, start_numpy_client, connection, and create_channel.

Reply:
Renamed `use_grpc_certificates` to `insecure` and updated the logic.